### PR TITLE
Senegal2018 revise

### PR DESCRIPTION
**Pull request by yining-sun**
- updated codes in 1.do- ANC
- updated codes in 2.do- delivery care - updated 11.do- sleeping under bednet. Count both treated and nontreated as ITN. Recode mother education = 6 as missing.
- Changed codes in 7.do child vaccination to ensure kids 15-23 months old receiving measles2 are coded as measles1.
- Revised comments on Child Illness.

### CODE REVIEW

File: 1_antenatal_care.do [hunk 1]
```diff
@@ -31,6 +31,15 @@
 	replace `var' = . if !inlist(`var',0,1)
 
 	 }
+	
+	if inlist(name, "Senegal2018") {
```
Comment (wengxyu1030) [inline 1_antenatal_care.do]: I would suggest to put the surveys in similar condition together to simply the command, for this, `if inlist(name,"Senegal2018", "Senegal2019", "Senegal2017")` would be better, could you modify？

File: 2_delivery_care.do [hunk 2]
```diff
@@ -23,6 +23,15 @@ gen country = regexs(1) if regexm(country_year, "([a-zA-Z]+)")
 	replace `var' = . if !inlist(`var',0,1)
 	
 	 }
+
```
Comment (wengxyu1030) [inline 2_delivery_care.do:26]: Could you follow my same suggestion that collapse the condition for surveys as below?
```suggestion
	 if inlist(name,"Senegal2018", "Senegal2019", "Senegal2017")

```

File: 7_child_vaccination.do [hunk 3]
```diff
@@ -14,6 +14,14 @@
 			replace c_measles = 1 if (h9 ==1 | h9 ==2 | h9 ==3)
 		}	
 		
+		if inlist(name, "Senegal2018") {
```
Comment (wengxyu1030) [inline 7_child_vaccination.do]: Please collapse the conditions as I suggested in the previous comments.

**Review by wengxyu1030 — CHANGES_REQUESTED** [review]
@yining-sun Thanks for the modifications, they look good. But I have put my comments where improvement could be made.

File: 1_antenatal_care.do (hunk 1 above)
Comment (yining-sun) [reply 1_antenatal_care.do]: @wengxyu1030 Dear Aline, I have updated the codes accordingly. Thanks for the comments!

**Comment by wengxyu1030** [comment]
Well done @yining-sun, I have reviewed your changes and it's good to go. I thereby merging it to the master file.

**Pull request merged**